LoopUI/Views/SettingsView.swift

```diff
@@ -0,0 +1,207 @@
+                supportSection
+            }
+            .listStyle(GroupedListStyle())
+            .navigationBarTitle(Text(LocalizedString("Settings", comment: "Settings screen title")))
+            .navigationBarBackButtonHidden(false)
+            .navigationBarHidden(false)
+            .navigationBarItems(trailing: dismissButton)
+            .environment(\.horizontalSizeClass, horizontalOverride)
+        }
+    }
+    
+}
+
+extension SettingsView {
+        
+    private var dismissButton: some View {
+        Button( action: { self.dismiss() }) {
+            Text("Done").bold()
+        }
+    }
+    
+    private var loopSection: some View {
+        Section (header: SectionHeader(label: viewModel.appNameAndVersion)) {
+            Toggle(isOn: $viewModel.dosingEnabled) {
+                Text(LocalizedString("Closed Loop", comment: "The title text for the looping enabled switch cell"))
+            }
+        }
+    }
+    
+    private var alertPermissionsSection: some View {
+        Section {
+            NavigationLink(destination:
+                NotificationsCriticalAlertPermissionsView(
+                    viewModel: NotificationsCriticalAlertPermissionsViewModel()))
+            {
+                HStack {
+                    Text(LocalizedString("Alert Permissions", comment: "Alert Permissions button text"))
+                    if viewModel.showWarning {
+                        Spacer()
+                        Text(LocalizedString("⚠️", comment: "Warning symbol"))
+                    }
+                }
+            }
+        }
+    }
+        
+    private var therapySettingsSection: some View {
+        Section (header: SectionHeader(label: LocalizedString("Configuration", comment: "The title of the Configuration section in settings"))) {
+            NavigationLink(destination: Text("Therapy Settings")) {
+                LargeButton(action: { },
+                            includeArrow: false,
+                            image: Image(bundleString: "Therapy Icon"),
+                            label: LocalizedString("Therapy Settings", comment: "Title text for button to Therapy Settings"),
+                            details: LocalizedString("Diabetes Treatment", comment: "Descriptive text for Therapy Settings"))
+            }
+        }
+    }
+    
+    private var deviceSettingsSection: some View {
+        Section {
+            pumpSection
+            cgmSection
+        }
+    }
+    
+    private var pumpSection: some View {
+        if viewModel.pumpManagerSettingsViewModel.isSetUp {
+            // TODO: this "dismiss then call onTapped()" here is temporary, until we've completely gotten rid of SettingsTableViewController
+            return LargeButton(action: { self.dismiss(); self.viewModel.pumpManagerSettingsViewModel.onTapped() },
+                               image: Image(uiImage: viewModel.pumpManagerSettingsViewModel.image, bundleString: "Omnipod"),
```

Comment:
Do you have a plan to replace the hardcoded "Omnipod" here and below?

Reply:
Yes: the plan is I am waiting for the answer to [my question in the Figma](https://www.figma.com/file/7da2yRPMUEdQnaQq3KvgZq?node-id=969:41662#31498363) file.